pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/module/AsciidocContentRenderingServlet.java

```diff
@@ -57,6 +57,7 @@ protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse r
         Optional<Content> content;
         if (draft) {
             content = module.getDraftContent(LocaleUtils.toLocale(locale));
+            log.info("asciidoctorservlet content");
```

Comment:
You shouldn't need to touch this file. I am 90% sure that this servlet is unused in our project... but no one has gotten around to deleting it.

Reply:
I might have to delete this